modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,56 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects_{context}"]
+= Enabling monitoring of event listeners for Triggers for user-defined projects
+
+As a cluster administrator, to gather event listener metrics for the `Triggers` service in a user-defined project and display them in the {product-title} web console, you can create a service monitor for each event listener. On receiving a HTTP request, event listeners for the `Triggers` service return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`.
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console.
+* You have installed the {pipelines-title} Operator.
+* You have enabled monitoring for user-defined projects.
```

Comment:
@sounix000 Do we need to provide any link reference here?

Reply:
We cannot provide direct inline links due to limitations set by the larger OpenShift docs org. I have given one link in the **Additional resources** section immediately after the procedure, and some other relevant links are in the bigger Additional resources section at the bottom of the page.

Reply:
Ack